jablib/src/main/java/org/jabref/logic/util/MscCodeUtils.java

```diff
@@ -0,0 +1,52 @@
+package org.jabref.logic.util;
+
+import java.io.IOException;
+import java.net.URL;
+import java.util.Map;
+import java.util.Optional;
+
+import org.jabref.logic.shared.exception.MscCodeLoadingException;
+
+import com.fasterxml.jackson.core.JsonParseException;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.JsonMappingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.common.collect.HashBiMap;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class MscCodeUtils {
+    private static final Logger LOGGER = LoggerFactory.getLogger(MscCodeUtils.class);
+
+    /** Load MSC codes and descriptions from a JSON resource URL into a HashMap
```

Comment:
Strangly formatted.

Either use markdown javadoc - or place the beginning of the description to the next line.

```suggestion
    /** 
     * Load MSC codes and descriptions from a JSON resource URL into a HashMap
```